.azure-pipelines/baseline_test/baseline.test.template.yml

```diff
@@ -120,14 +113,13 @@ jobs:
         BUILD_REASON: "BaselineTest"
         RETRY_TIMES: "0"
         STOP_ON_FAILURE: "False"
+        TEST_PLAN_NUM: $(BASELINE_MGMT_PUBLIC_MASTER_TEST_NUM)
```

Comment:
Please confirm that other feature branches like 202305 and nightly test plan would be affected by the change of .azure-pipelines/run-test-elastictest-template.yml

Reply:
confirmed, with the change, 202305 and nightly test plan could work well